usaspending_api/recipient/migrations/0012_summaryawardrecipient.py

```diff
@@ -16,9 +16,9 @@ class Migration(migrations.Migration):
             name='SummaryAwardRecipient',
             fields=[
                 ('award_id', models.TextField(primary_key=True, serialize=False)),
-                ('action_date', models.DateField(blank=True)),
-                ('recipient_hash', models.UUIDField(null=True)),
-                ('parent_recipient_unique_id', models.TextField(null=True)),
+                ('action_date', models.DateField(blank=True, db_index=True)),
```

Comment:
missing model.py edits?

Reply:
Huh, curious how that slipped. Great catch.